start/manage/index.php

```diff
@@ -0,0 +1,31 @@
+<?php
+require 'header.php';
+
+use Service\Container;
+
+$container = new Container($configuration);
+$shipLoader = $container->getShipLoader();
+$ships = $shipLoader->getShips();
+
+
+?>
+
+<div class="container">
+    <div class="row">
+        <?php foreach ($ships as $ship): ?>
+            <div class="col-sm-6">
+                <h2><a href="/manage/show.php?id=<?php echo $ship->getId();?>"><?php echo $ship->getName();?></a></h2>
+
+                <?php //var_dump($ship->getImage());die;?>
+                <?php if ($ship->getImage() == ""){ ?>
```

Comment:
Remove commented code above and I think this should be checking for null instead of and empty string